plugins/window_size/lib/src/window_size_channel.dart

```diff
@@ -163,6 +172,16 @@ class WindowSizeChannel {
     ]);
   }
 
+  /// Sets the title of the window containing this Flutter instance.
+  void setWindowTitle(String title) async {
+    await _platformChannel.invokeMapMethod(_setWindowTitleMethod,title);
```

Comment:
Nit: space after `,`

Reply:
Ran `dartfmt` over the file to tidy everything up.